crates/api_models/src/enums.rs

```diff
@@ -552,6 +552,7 @@ pub enum RoutableConnectors {
     Globalpay,
     Klarna,
     Payu,
+    Rapyd,
```

Comment:
How is this change related to this fix ?

Reply:
its not just saw Rapyd missing